src/helpers/money.ts

```diff
@@ -5,7 +5,7 @@ export const convertToCents = (dollars?: number | string): number => {
   return Math.round(Number(dollars) * 100) // Round to avoid #'s like 7001.000000000001
 }
 
-export const formatMoney = (cents: number): string => {
+export const formatMoney = (cents: number | undefined): string => {
   if (!cents || !Number.isFinite(+cents)) {
```

Comment:
Good catch on updating this to be in line with our actual usage of it (which does include passing it `undefined` sometimes).

However, if an `undefined` is passed, I think it may be more appropriate to return an empty string, rather than explicitly saying `$0.00` when (perhaps) some data simply hasn't loaded yet. Thoughts?
```suggestion
export const formatMoney = (cents: number | undefined): string => {
  if (cents === undefined) {
    return ''
  }
  if (!cents || !Number.isFinite(+cents)) {
```

Reply:
I agree, particularly if it's only `undefined` during page loading. In any case, showing "$0.00" would be misleading.